API/Services/SeriesService.cs

```diff
@@ -505,8 +504,7 @@ public async Task<SeriesDetailDto> GetSeriesDetail(int seriesId, int userId)
         } else
         {
             retChapters = chapters
-                .Where(ShouldIncludeChapter)
-                .OrderBy(c => float.Parse(c.Number), new ChapterSortComparer());
```

Comment:
The list of chapters has already been sorted and the Where filter does not affect it.